internal/database/pg.go

```diff
@@ -223,7 +223,9 @@ func (db *PGDatabase) GetBookmarks(ctx context.Context, opts GetBookmarksOptions
 	if opts.Keyword != "" {
 		query += ` AND (
 			url LIKE :lkw OR
-			MATCH(title, excerpt, content) AGAINST (:kw IN BOOLEAN MODE)
```

Comment:
It seems that MATCH ... AGAINST is a mysql syntax for text searching. In postgres, I don't find such simple way to implement text matching so I use LIKE multi times as a workround.

Reply:
Yeah, we need to implement FTS into PSQL, I need to take a look at #276 first in order to decide an engine.